docs/sample_config.yaml

```diff
@@ -67,12 +67,11 @@ pid_file: DATADIR/homeserver.pid
 #
```

Comment:
Please run `scripts-dev/generate_sample_config` to fix this file and the logging sample config.

Reply:
Looks correct to me! Also thank you for fixing a typo in the log config :)